src/Feeds.php

```diff
@@ -127,8 +128,7 @@ public static function is_local_feed_registered( $merchant_id ) {
 		$configured_path = dirname( $feed->location_config->full_feed_fetch_location );
 		$local_path      = dirname( $config['feed_url'] );
 		$local_country   = Pinterest_For_Woocommerce()::get_base_country() ?? 'US';
-		$local_locale    = str_replace( '_', '-', determine_locale() );
-
+		$local_locale    = LocaleMapper::get_locale_for_api();
```

Comment:
Should we have a `try`/`catch` block around this to handle the possible `PinterestApiLocaleException` exception? It looks like only one place where this method is called is surrounded by a try/catch block.

Reply:
It is a bit convoluted but I think that at this stage all possible occurrences of this exception should be handled by one or another `try/catch` block.

We should probably create a separate issue and streamline exception handling because it is right now all over the place - but that should be a separate PR IMO.